src/phoenix/code_examples/llm_evaluator_subclass.py

```diff
@@ -0,0 +1,40 @@
+from types import MappingProxyType
+from typing import Any, Optional
+
+from phoenix.datasets.evaluators.base import LLMEvaluator
+from phoenix.datasets.types import (
+    EvaluationResult,
+    ExampleInput,
+    ExampleMetadata,
+    ExampleOutput,
+    TaskOutput,
+)
+
+
+class MyLLMEvaluator(LLMEvaluator):
```

Comment:
I think the problem with this code snippet is it also seems extremely complicated when I think we decided the decorator is an okay approach for now and learn from it.

I think if we go the sub-classing route we should at least have like a prompt template in there. Otherwise this is not super informative because it doesn't work out of the box.